.circleci/config.yml

```diff
@@ -32,7 +32,7 @@ jobs:
                     python -m venv venv || virtualenv venv
                     . venv/bin/activate
                     pip install -r dev-requirements.txt --quiet
-                    git clone --depth 1 git@github.com:plotly/dash.git dash-main
+                    git clone --depth 1 -b loosen-testing-reqs git@github.com:plotly/dash.git dash-main
```

Comment:
we can revert these branches and merge this PR even before https://github.com/plotly/dash/pull/1506 merges, because the changes here are fine with `dash@dev`